docs/tutorials/cargo_tests/baked/Cargo.toml

```diff
@@ -7,7 +7,7 @@
 ##########################################
 
 [package]
-name = "demo"
+name = "demo_baked"
 version = "0.1.0"
```

Comment:
Nit: we bump version numbers before publishing, so new crates should start at `0.0.0`. Also all our no-publish crates are set to `0.0.0`.